### DISCUSSION TITLE
Support record collection name in db.bind_vars for java mongodb-agent

### DISCUSSION THREAD

**Discussion by suninuni** [apache/skywalking#9313] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

db.bind_vars is correctly show but not record the collection name, which is may be useful.

![image](https://user-images.githubusercontent.com/7345529/177447005-ac4e38ab-aec7-42ba-a134-b048d0aa4a75.png)

Another questions is why only the mongodb-3.x and mongodb-4.x use db.bind_vars? I think they should be db.statement according to the development guide. https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/java-plugin-development-guide/#tag-keys-dbstatement-and-dbtype

![image](https://user-images.githubusercontent.com/7345529/177448294-5fd6d598-70cb-43ab-98fb-f38acc64d653.png)

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Please use GitHub correctly. Discussion is the place to discuss uncertain features.

**Comment by wu-sheng** [comment]
I think `db.statement` or `db.bind_vars` is determined by the plugin. `db.statement` is only required when the statement should be sampled according to slow threshold. In your case, it is very clear, it is not a statement, it is just a collection of parameters in MongoDB operation.
So, I believe the current tag use is correct.

**Reply by wu-sheng** [reply]
collection operation actually is the endpoint name, a.k.a. span's operation name.

**Reply by suninuni** [reply]
So it' seems there is no existed tag for the collection name.

For example, there is a mongodb statement `db.getSiblingDB("test").account.find({id:""})`, the span's endpoint whill be `MongoDB/FindOperation`, `db.bind_vars` will be `{id:""}`, and there should be a new tag to record the collection name `account` and the database name `test`?

**Reply by wu-sheng** [reply]
I was thinking to add a `db.table` as we may analyze table-based metrics. But, after evaluating https://github.com/apache/shardingsphere/issues/8208, it seems the performance is not enough.
So, we canceled that plan. If you are willing to contribute, I think we could accept this new tag.
